src/test/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/integration/EstablishmentsIntegrationTest.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.digitalprisonreportingmi.integration
+
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.data.Establishment
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.data.EstablishmentRepository
+import java.util.stream.IntStream
+
+class EstablishmentsIntegrationTest : IntegrationTestBase() {
+
+  @Autowired
+  lateinit var establishmentRepository: EstablishmentRepository
+
+  @Test
+  fun `Establishments count returns expected value`() {
+    val expectedCount = 123
```

Comment:
Just thinking about the test runtime. Maybe just insert 2 records and check?

Reply:
It's a fair point. Currently it is'n't an issue:

> SUCCESS: Executed 10 tests in 8.8s

But that will build up over time.

Reply:
This has now been changed to 2.